Adjustmets WalletHeaderView

gemdev111 opened the pull request:
Updates Round button & Buttons view
Adjusted AssetsScene & WalletScene with updated WalletHeaderView

close #125

images:
<table>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/4c299354-4240-4cdf-ab1b-9d4a6d4a49c7" style="width: 45%;" /></td>
    <td><img src="https://github.com/user-attachments/assets/2974e0cb-dcad-44d8-9ac1-fbfa9e20758d" style="width: 45%;" /></td>
  </tr>
</table>

gemdev111 committed:
Adjustmets WalletHeaderView

Updates Round button & Buttons view
Adjusted AssetsScene & WalletScene with updated WalletHeaderView

close #125

gemdev111 committed:
Merge branch 'main' into walletheader

gemdev111 committed:
Removed debug helpers

gemcoder21 commented on Packages/Components/Sources/Buttons/RoundButton.swift:
```diff
@@ -18,30 +24,49 @@ public struct RoundButton: View {
         self.image = image
         self.title = title
+
+        if let windowScene = UIApplication.shared.connectedScenes.first as? UIWindowScene {
```
can we not use `UIWindowScene` and find other solutions?

gemdev111 committed:
Merge branch 'main' into walletheader

gemdev111 committed:
submodules updates

gemdev111 committed:
Merge branch 'main' into walletheader

gemdev111 committed:
Removed UIScene checkers
Added DeviceLayoutState - responsible for getting device layout info like: size, orientation, safe area, ipad compactness checking
